lib/membrane_rtc_engine/endpoints/webrtc/utils.ex

```diff
@@ -13,4 +13,85 @@ defmodule Membrane.RTC.Engine.Endpoint.WebRTC.Utils do
     <<_size0::3, _h::1, _ver::3, p::1, _size1::8, _size2::8, _rest::binary()>> = payload
     p == 0
   end
+
+  # This is rewritten from galene
+  # https://github.com/jech/galene/blob/6fbdf0eab2c9640e673d9f9ec0331da24cbf2c4c/codecs/codecs.go#L119
+  # and based on https://datatracker.ietf.org/doc/html/rfc6184#section-5
+  @spec is_h264_keyframe(binary()) :: boolean()
```

Comment:
Handling H264 RTP packetization is implemented in membrane_rtp_h264_plugin, while keyframe detection in Membrane.H264.FFmpeg.Parser. We should check if we could possibly use/extend them.

Reply:
It's only partially native :P https://github.com/membraneframework/membrane_h264_ffmpeg_plugin/blob/master/lib/membrane_h264_ffmpeg/parser/nalu.ex

Reply:
> The perspective that we have to include whole ffmpeg h264 repo just to use one function doesn't look good to me.

Then we may move that to membrane_h264_format. The point is, someone will surely need that at some point and I doubt they'll look at webrtc utils :P I'd at least make this `Membrane.RTP.H264.is_key_frame?`